source/designers-guide/smarty-plugins/index.md

```diff
@@ -0,0 +1,257 @@
+
+Example: *example of a call to instant downloads*
+``` 
+{* A link to instant downloads *}
+<a href="{url controller='account' action='downloads'}" title="Open instant downloads">
+	Open instant downloads
+</a>
+``` 
+ 
+You also have the option to pass on to the controller parameters. For this example, we'll extend the call to instant downloads with the parameter sParam with the value test.
+
+Exmaple: *call to instant download with parameter*
+``` 
+{* A link to instant downloads *}
+<a href="{url controller='account' action='downloads' sParam='test'}" title="Open instant downloads with parameter">
+	Open instant downloads with parameter
+</a>
+``` 
+ 
+The syntax of the url plugin is as follows:
+ 
+ 
+Example: *Syntax url plugin*
+``` 
+{* Syntax *}
+{url controller='[CONTROLLERNAME]' action='[ACTIONNAME]' [MORE PARAMETERS='PARAMETERVALUE']}
+``` 
+ 
+The plugin also automatically builds up the SEO links if the corresponding plugin is installed.
+
+## Date plugin ##
+
+The date plugin is used to format date and time information. To this end, you have the ability to use a variety of formats. The syntax of the date plugin is as follows:
+
+Example: *date plugin syntax*
+``` 
+{* Syntax *}
+{[WERT]|date:[FORMAT]:[TYP]}
+``` 
+ 
+The plugin has the following format types, which are based on the Zend Framework:
+
+<div class="alert alert-info">
+	Note: Due to localization, the following date formats may differ slightly from the examples given.
+</div>
+
+Date:
+
+- DATE_FULL - full date e.g., "Thursday, November 4, 2010"'
+- DATE_LONG - DATE_LONG - long date e.g., "4th of November 2010"
+- DATE_MEDIUM - normal date e.g., "04/11/2010"
+- DATE_SHORT - abbreviated date e.g., "11.04.10"'
+
+Time:
+
+- TIME_FULL - full time e.g., "1:55:52 pm Europe/Berlin"
+- TIME_LONG - long time e.g., "1:55:52 pm CET"
+- TIME_MEDIUM - normal time e.g., "1:55:52 pm"
+- TIME_SHORT - abbreviated time e.g., "1:55 pm"
+
+Date/time:
+
+- DATETIME_FULL - full date with time e.g., "Thursday, November 4, 2010 1:55:52 pm Europe / Berlin"
+- DATETIME_LONG - long date with time e.g., "November 4, 2010 1:55:52 pm CET"
+- DATETIME_MEDIUM - normal date with time e.g., "11/04/2010 1:55:52 pm"
+- DATETIME_SHORT - abbreviated date with time e.g., "11/04/10 1:55 pm"
+
+Miscellaneous:
+
+- ISO_8601 - date according to ISO 8601 e.g., "2010-11-04T13:55:52+01:00"
+- RFC_2822 - date according to RFC 2822 e.g., "Thu, 04 Nov 2010 13:55:52 +0100"
+- TIMESTAMP - UNIX time e.g., "1288875352"
+- ATOM - date according to ATOM e.g., "2010-11-04T13:55:52+01:00"
+- RSS - date for RSS feeds e.g., "Thu, 04 Nov 2010 13:55:52 +0100"
+- COOKIE - date for cookies e.g., "Thursday, 04-Nov-10 1:55:52 p.m. Europe / Berlin"
+- W3C - date for HTML or HTTP W3C e.g., "2010-11-04T13:55:52+01:00"
+
+The following types can be defined for the plugin:
+
+- ISO - ISO format for date formatting
+- PHP - PHP's date() function for date formatting
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+
+## Rewrite plugin ##
```

Comment:
the rewrite plugin does not exist anymore. I think an old commit is still here :blush: